decidim-process-extended/decidim-process-extended.gemspec

```diff
@@ -16,6 +16,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
   s.add_dependency "rails", ">= 5.2", "< 6.0.x"
+  s.add_dependency "decidim-core"
 
   s.add_development_dependency "sqlite3"
```

Comment:
we should remove this dependencies to sqlite3, as Decidim depends directly from PostgreSQL.